### PR TITLE
DT Exchange iOS SDK version 8.1.6 (updated)

### PR DESCRIPTION
**Pull request by TomerFyber**
- updated google adapters to be compatible with DT Exchange iOS SDK version 8.1.6: modified ad targeting data assignment way according to SDK changes (global SDK assignment instead of per-request);
- fixed identation

### CODE REVIEW

**Review by 2energycell — APPROVED** [review]

**Comment by TomerFyber** [comment]
@joshh-devrel Hey, can you please review and merge?

**Comment by joshh-devrel** [comment]
@TomerFyber Can you revert all the indentation changes? The format will be handled by our tool internally.

**Comment by TomerFyber** [comment]
@joshh-devrel Reverted the indentation changes. Thanks!

**Comment by joshh-devrel** [comment]
@TomerFyber Can you sign the CLA as well? Thank you!

**Comment by TomerFyber** [comment]
@joshh-devrel Done :)

File: adapters/Fyber/FyberAdapter/GADMAdapterFyberBannerAd.m [hunk 1]
```diff
@@ -124,9 +124,9 @@ - (void)loadBannerAd {
 
   IAAdRequest *request =
       GADMAdapterFyberBuildRequestWithSpotIDAndAdConfiguration(spotID, _adConfiguration);
+    IASDKCore.sharedInstance.mediationType = IAMediationAdMob.new;
```
Comment (joshh-devrel) [inline adapters/Fyber/FyberAdapter/GADMAdapterFyberBannerAd.m:127]: @TomerFyber Can you update to use `alloc] init];` instead?
The same applies to the other files.

**Pull request merged**